### DISCUSSION TITLE
The order in fparam.raw

### DISCUSSION THREAD

**Discussion by phyoung123** [deepmodeling/dpgen#921] [Q&A]
I want to train a model for different tempeartures such as 3000K,4000K, 5000K, the raw files of energy and force are extracted by this code

`from dpdata import LabeledSystem
from glob import glob
outcar_file = glob('./*/OUTCAR')

total_system=LabeledSystem()
for f in outcar_file:
    ls=LabeledSystem(f, fmt='vasp/outcar')
    total_system.append(ls)
total_system.shuffle()

split_num = int(len(total_system) * 0.6)      # should be convert to int
total_system[:split_num].to_deepmd_npy('./data/training_set', set_size=2000)
total_system[:split_num].to_deepmd_raw('./data/training_set', set_size=2000)
total_system[split_num:].to_deepmd_npy('./data/validation_set', set_size=500)
total_system[split_num:].to_deepmd_raw('./data/validation_set', set_size=500)`

As you can see the energy and force are shuffled randomly but the fparam.raw must be prepared manually follow the frame order, so the order in these files are not consistance. Does it matter?

Thank you in advance.

**Comment by AnguseZhang** [comment]
Maybe you can change your scripts such that the order of `fparam` is consistent with your original data .

**Reply by phyoung123** [reply]
Meanwhile, I have another question, How can I considering different temperatures during Labeling step
